apps/src/applab/RedirectDialog.jsx

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Dialog, {Body} from '@cdo/apps/templates/Dialog';
+import DialogFooter from '../templates/teacherDashboard/DialogFooter';
+import Button from '../templates/Button';
+import i18n from '@cdo/locale';
+
+export default class RedirectDialog extends React.Component {
+  static propTypes = {
+    url: PropTypes.string.isRequired,
+    approved: PropTypes.bool,
+    isOpen: PropTypes.bool
+  };
+
+  state = {
+    isOpen: this.props.isOpen
+  };
+
+  closeDialog = () => {
+    this.setState({isOpen: false});
+  };
+
+  render() {
+    let title, body, footer;
+
+    if (this.props.approved) {
+      title = i18n.redirectTitle();
+      body = (
+        <div>
+          <h2>{i18n.redirectConfirm({url: this.props.url})}</h2>
+          <p>
+            {i18n.redirectExplanation()}
+            <span>
+              <a target="_blank" href={'https://studio.code.org/report_abuse'}>
+                {i18n.reportAbuse()}
+              </a>
+            </span>
+          </p>
+        </div>
+      );
+      footer = (
+        <DialogFooter>
+          <Button
+            onClick={this.closeDialog}
+            text={i18n.goBack()}
+            color={Button.ButtonColor.gray}
+          />
+          <Button
+            href={this.props.url}
+            target={'_blank'}
+            text={i18n.continue()}
+            color={Button.ButtonColor.orange}
```

Comment:
I think these are already styled somewhere. I'm looking to see if I can find it, but in the meantime, it's something like continue-button. (From your screenshot, I think this is currently missing some style elements such as the curved corners)

Reply:
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/DialogButtons.jsx

https://github.com/code-dot-org/code-dot-org/blob/ecdbc30e18d75c522f4450351875851efb8be3ee/apps/src/submitHelper.js#L108

Reply:
Good catch! - I was resisting using DialogButtons since it relies on deprecated content, so I dove deeper into our Button component and discovered that there was an error in the css that was preventing the rounded corners from displaying. I updated the screenshots, so let me know if those appear as expected.